besu/src/main/java/org/hyperledger/besu/controller/BesuControllerBuilder.java

```diff
@@ -159,8 +159,8 @@
     return this;
   }
 
-  public BesuControllerBuilder<C> isPruningEnabled(final boolean pruningEnabled) {
-    this.isPruningEnabled = pruningEnabled;
+  public BesuControllerBuilder<C> pruningEnabled(final boolean pruningEnabled) {
+    this.pruningEnabled = pruningEnabled;
```

Comment:
(nit) I think `isPruningEnabled` is more explicit and more consistent with codebase conventions